contracts/BlankArt.sol

```diff
@@ -34,6 +34,8 @@ contract BlankArt is ERC721, EIP712, ERC721URIStorage {
     uint256 public maxTokenSupply;
     // cost to mint during the public sale
     uint256 public mintPrice;
+    // Enables/Disables voucher redemption
+    bool public active;
```

Comment:
nit: should we call this "voucherRedemptionActive" or something slightly more descriptive?

Reply:
Great feedback! I ended up using it for both redemption and minting, so at a minimum need to update the comment. If we want to restrict it to only redemption we can make the name more specific too!